src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.Edit.cshtml

```diff
@@ -40,30 +43,87 @@
         </div>
 
         <div class="mt-2">
-            <div class="btn btn-warning btn-sm" id="generatePassword">@T["Generate password"]</div>
+            <div class="btn btn-warning btn-sm" id="generatePassword" data-password-length="@IdentityOptions.Value.Password.RequiredLength">@T["Generate password"]</div>
             <div class="btn btn-info btn-sm" id="copyPassword">@T["Copy password"]</div>
         </div>
     </div>
-
     <script at="Foot" asp-name="userFieldsPassword">
         document.addEventListener('DOMContentLoaded', function() {
-            const strongPasswordLength = 12;
             let password = document.getElementById('@Html.IdFor(p => p.Password)');
```

Comment:
Can we depends on jQuery and simplify accessing to the elements as well as data attributes like usually did?

Reply:
After bootstrap 5 upgrade, I think we should be encouraging to not use jquery. IMO, vanilla-JS here is much better than having to depend on another framework just to save very few lines of code.

Reply:
You could file a new issue for that

Reply:
No JQuery if it is not necessary.